scripts/drone/steps/lib.star

```diff
@@ -464,7 +466,7 @@ def test_a11y_frontend_step(ver_mode, edition, port=3001):
             'HOST': 'end-to-end-tests-server' + enterprise2_suffix(edition),
             'PORT': port,
         },
-        'failure': 'ignore',
+        'failure': failure,
```

Comment:
am i understanding correctly that this will fail on PRs, but not fail on main branch builds? is this just because it's still a bit flakey and we don't want to impact the main branch? 🤔

Reply:
You're correct, but it's not because of any flakiness it's because you can't run `pa11y-ci` to just create the report that we need to update the frontend metrics, it will always fail if there are errors. I created this issue: https://github.com/pa11y/pa11y-ci/issues/161 that I hope explains it.